cogboard-webapp/src/components/widgets/dialogFields/TimestampInput.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { StyledValidationMessages } from '../../WidgetForm/styled';
+import { hasError } from '../../../utils/components';
+import MomentUtils from '@date-io/moment';
+import moment from 'moment-timezone';
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
+import { prepareChangeEvent } from './helpers';
+import CloseIcon from '@material-ui/icons/Close';
+import {
+  DatePickerWrapper,
+  StyledDateTimePicker,
+  StyledIconButton
+} from './styled';
+
+const TimestampInput = ({
+  error,
+  dataCy,
+  value,
+  label,
+  onChange,
+  ...other
+}) => {
+  const handleChange = date =>
+    onChange(prepareChangeEvent(date && date.utc().unix(), 'number?'));
+
+  return (
+    <DatePickerWrapper>
+      <MuiPickersUtilsProvider utils={MomentUtils}>
+        <StyledDateTimePicker
+          InputLabelProps={{
+            shrink: true
+          }}
+          margin="normal"
+          label={label}
+          value={
+            Number.isInteger(value)
+              ? moment
+                  .utc(value * 1000)
+                  .local()
+                  .format()
+              : null
+          }
+          error={hasError(error)}
+          onChange={handleChange}
+          format="YYYY-MM-DD HH:mm"
```

Comment:
czy ten format nie jest już gdzieś zdefiniowany? jeśli nie, to proponuję zdefiniować, bo to chyba standardowy format i wypada, żeby był reużywalny